pkg/deploy/lattice/service_network_manager_test.go

```diff
@@ -284,18 +287,267 @@ func Test_CreateServiceNetwork_MeshAlreadyExist_ServiceNetworkVpcAssociationStat
 			SvcNetwork: item,
 			Tags:       nil,
 		}, nil)
+	mockLattice.EXPECT().GetServiceNetworkVpcAssociationWithContext(ctx, gomock.Any()).Return(&vpclattice.GetServiceNetworkVpcAssociationOutput{
+		ServiceNetworkArn:  &meshArn,
+		ServiceNetworkId:   &meshId,
+		ServiceNetworkName: &name,
+		Status:             aws.String(vpclattice.ServiceNetworkVpcAssociationStatusActive),
+		VpcId:              &vpcId,
+	}, nil)
 	mockLattice.EXPECT().ListServiceNetworkVpcAssociationsAsList(ctx, gomock.Any()).Return(statusServiceNetworkVPCOutput, nil)
+	mockLattice.EXPECT().UpdateServiceNetworkVpcAssociationWithContext(ctx, gomock.Any()).Times(0)
 	mockCloud.EXPECT().Lattice().Return(mockLattice).AnyTimes()
 
 	meshManager := NewDefaultServiceNetworkManager(mockCloud)
-	resp, err := meshManager.Create(ctx, &meshCreateInput)
+	resp, err := meshManager.CreateOrUpdate(ctx, &meshCreateInput)
 
 	assert.Nil(t, err)
 	assert.Equal(t, resp.ServiceNetworkARN, meshArn)
 	assert.Equal(t, resp.ServiceNetworkID, meshId)
 }
 
-func Test_CreateServiceNetwork_MeshAlreadyExist_AssociateToNotAssociate(t *testing.T) {
```

Comment:
Did the unit test name changes: from ` _CreateServiceNetwork_` to ` _CreateOrUpdateServiceNetwork_` i.e, from `Test_CreateServiceNetwork_MeshAlreadyExist_AssociateToNotAssociate ` to `Test_CreateOrUpdateServiceNetwork_MeshAlreadyExist_AssociateToNotAssociate`